docs/source/api-reference/array-manipulation-routines.rst

```diff
@@ -28,6 +24,7 @@ Transpose-like operations
    :toctree: generated/
 
    swapaxes
+   ndarray.T
```

Comment:
`ndarray.T` doesn't show up at all when I compile locally.

Reply:
Good catch. I know `BlockArray.T` works, but I'm not sure where it's defined. @elibol do you know? Once I know where it's defined, I can add it to the docs